pureskillgg_dsdk/tome/curator_test.py

```diff
@@ -28,14 +28,15 @@ def create_curator_instance(tmp_path):
 
 
 def create_header(curator):
-    return curator.create_header_tome(path_depth=1)
+    return curator.create_header_tome()
 
 
 def create_header_and_subheader(curator):
     create_header(curator)
     return curator.create_subheader_tome(
         sub_header_name,
-        lambda df: df["key"] != "0F9P4Bte2Z1GLiuOsryY",
+        lambda df: df["key"]
+        != r"csds\2022\05\10\0a1effe9-262a-4828-bd4e-8a4265905905\csds",
```

Comment:
Why `r"` here? And they key should be normalized to `/` right? Why is it `\` here?

Reply:
why is it working locally for me lol

Reply:
fixed w/ proper key from manifest